python/lsst/afw/fits/fitsLib.py

```diff
@@ -1 +1,3 @@
 from __future__ import absolute_import
```

Comment:
Please update to `from __future__ import absolute_import, division, print_function`

both here and in all other .py files you modified. It is our standard, and division is helpful in avoiding subtle bugs that may only show up in Python 2.7 or Python 3.